testData/generator/StubFallback.expected.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2011-2025 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+ */
```

Comment:
the test fails because of the copyright and the last line

<img width="1312" height="672" alt="Screenshot 2025-11-28 at 19 58 07" src="https://github.com/user-attachments/assets/f01982af-fe62-4e0c-8f4a-f8e4ad2d8c89" />